content/community.html

```diff
@@ -182,7 +182,7 @@ <h1>Acknowledgements</h1>
       <a href="https://www.docker.com/">Docker</a> and <a href="http://www.boxever.com/">Boxever</a>.
     </p>
     <p>
-      Special thanks to <a href="https://digitalocean.com/">DigitalOcean</a> for providing hosting resources.
+      Special thanks to <a href="https://digitalocean.com/">DigitalOcean <img src="/assets/digitalocean_logo.svg" alt="DigitalOcean logo" width="200"></a> for providing hosting resources.
```

Comment:
Width 200 takes up the whole page width, which is a bit much.

```suggestion
      Special thanks to <a href="https://digitalocean.com/">DigitalOcean <img src="/assets/digitalocean_logo.svg" alt="DigitalOcean logo" width="50"></a> for providing hosting resources.
```

Reply:
Actually, I'm not sure `width` works correctly with SVGs.

```suggestion
      Special thanks to <a href="https://digitalocean.com/">DigitalOcean <img src="/assets/digitalocean_logo.svg" alt="DigitalOcean logo" style="width:200px"></a> for providing hosting resources.
```